armi/reactor/components/component.py

```diff
@@ -783,6 +783,10 @@ def getBoundingCircleOuterDiameter(self, Tc=None, cold=False):
         """Abstract bounding circle method that should be overwritten by each shape subclass."""
         raise NotImplementedError
 
+    def getCircleInnerDiameter(self, Tc=None, cold=False):
+        """Abstract inner bounding circle method that should be overwritten by each shape subclass."""
```

Comment:
This needs a better doc string.  Mentioning intersecting corners would add value.

Reply:
Addressed https://github.com/terrapower/armi/pull/670/commits/7eb3cccc97bc6d135c73c9ab372f5a5627fc0585